create_playlist.py

```diff
@@ -1,31 +1,33 @@
 # step 1 sign into youtube
 # step 2 create a new playlist album
-# step 3 grab like song on youtube 
+# step 3 grab like song on youtube
 # step 4 search for song
 # step 5 add song into album
 
-# PYTHON IS AN INTERPRETTED LANGUAGE (COMPILE BUTTON) AND OBJECT ORIENTED. 
+# PYTHON IS AN INTERPRETTED LANGUAGE (COMPILE BUTTON) AND OBJECT ORIENTED.
 
-import json # format for storing and transporting data.  A key and value data format to transfer data
+import json  # format for storing and transporting data.  A key and value data format to transfer data
 import requests  # an HTTP library  (communication from user to code)
 import os
 
 
-from secret import spotify_user_id,spotify_token  #takes the variable we want from secret file
+# takes the variable we want from secret file
+from secret import spotify_user_id, spotify_token
 import google_auth_oauthlib.flow
 import googleapiclient.discovery
 import googleapiclient.errors
 import youtube_dl
- 
+
+
 class CreatePlaylist:
-    
+
     def _init_(self):
```

Comment:
this should be `__init__`, not `_init_` (double underscores). This type of method (and others, for example: `__add__`, `__len__` are often referred to as dunder methods (**d**ouble **under**score) or magic methods.

Reply:
just fixed that, Definitely never have heard of that term, Thanks for this correction!

Reply:
No problem! :smile:

Reply:
just as a fyi, you *might* create merge conflicts (you and I both change a line of code, and then git doesn't know which change to accept) if you make changes to your code **before** merging this pull request.